samples/cpp/common/utils/include/samples/classification_results.h

```diff
@@ -15,6 +15,7 @@
 #include <utility>
 #include <vector>
 
+#include "openvino/core/type/element_type_traits.hpp"
```

Comment:
is it not accessible from `openvino/openvino.hpp` ?
maybe we can add it there then?

Reply:
Great